trueos-handbook/introducing.rst

```diff
@@ -1083,10 +1197,14 @@ wireless devices. Verify the device is supported by clicking the
 an external wireless device, insert it before running the installer.
 
 Certain Broadcom devices, typically found in less expensive laptops, are
-buggy and can have lockups when in *DMA* mode. If the device freezes,
-try switching to *PIO* mode in the BIOS. Alternately, add
+buggy and lockup unexpectedly while in *DMA* mode. If the device
+freezes, try switching to *PIO* mode in the BIOS. Alternately, add
 :command:`hw.bwn.usedma=0` to :file:`/boot/loader.conf` and reboot to
-see if anything changes.
+see if that resolves the issue.
+
+.. note:: Some wifi adapters are not configured during installation,
+	  but after first boot. Just because an adapter does not show
+	  up during installation does not mean it's not supported.
```

Comment:
Too many "nots":
"does not mean it's not supported."
->
"does not mean it is unsupported."

Reply:
I swear I'm not a member of the Department of Redundancy Department. ;)